README.md

```diff
@@ -70,6 +70,61 @@ the user needs to provide very minimal configuration for monitoring any type of
 
 Luminaire can also monitor a set of data points over windows of time instead of tracking individual data points. This approach is well-suited for streaming use cases where sustained fluctuations are of greater concern than individual fluctuations. See [anomaly detection for streaming data](https://zillow.github.io/luminaire/tutorial/streaming.html) for detailed information.
 
+## Examples
+
```

Comment:
In this example, should we add some example for input and output generated as comments?

Reply:
I just wanted to have it short and simple. We can definitely add more stuffs but users can always go to our documentations to find more details.

Reply:
So, can you add comments to refer the docs for more details?

Reply:
I referred to the example section in the quick start and it also has a link to the main docs.